versioned_docs/version-8.6/self-managed/zeebe-deployment/configuration/logging.md

```diff
@@ -3,65 +3,53 @@ id: logging
 
-- `config/log4j2.xml` (applied by default)
-
-```xml
-<?xml version="1.0" encoding="UTF-8"?>
-<Configuration status="WARN" shutdownHook="disable">
-
-  <Properties>
-    <Property name="log.path">${sys:app.home}/logs</Property>
-    <Property name="log.pattern">%d{yyyy-MM-dd HH:mm:ss.SSS} [%X{actor-name}] [%t] %-5level %logger{36} - %msg%n</Property>
-    <Property name="log.stackdriver.serviceName">${env:ZEEBE_LOG_STACKDRIVER_SERVICENAME:-}</Property>
-    <Property name="log.stackdriver.serviceVersion">${env:ZEEBE_LOG_STACKDRIVER_SERVICEVERSION:-}</Property>
-  </Properties>
-
-  <Appenders>
-    <Console name="Console" target="SYSTEM_OUT">
-      <PatternLayout
-        pattern="${log.pattern}"/>
-    </Console>
-
-    <Console name="Stackdriver" target="SYSTEM_OUT">
-      <StackdriverLayout serviceName="${log.stackdriver.serviceName}"
-        serviceVersion="${log.stackdriver.serviceVersion}" />
-    </Console>
-
-    <RollingFile name="RollingFile" fileName="${log.path}/zeebe.log"
-      filePattern="${log.path}/zeebe-%d{yyyy-MM-dd}-%i.log.gz">
-      <PatternLayout>
-        <Pattern>${log.pattern}</Pattern>
-      </PatternLayout>
-      <Policies>
-        <TimeBasedTriggeringPolicy/>
-        <SizeBasedTriggeringPolicy size="250 MB"/>
-      </Policies>
-    </RollingFile>
-  </Appenders>
-
-  <Loggers>
-    <Logger name="io.camunda.zeebe" level="${env:ZEEBE_LOG_LEVEL:-info}"/>
-
-    <Logger name="io.atomix" level="${env:ATOMIX_LOG_LEVEL:-warn}"/>
-
-    <Root level="info">
-      <AppenderRef ref="RollingFile"/>
-
-      <!-- remove to disable console logging -->
-      <AppenderRef ref="${env:ZEEBE_LOG_APPENDER:-Console}"/>
-    </Root>
-  </Loggers>
-
-</Configuration>
-```
+[You can find the default log4j2.xml used by the application here](https://github.com/camunda/camunda/blob/stable/8.6/dist/src/main/config/log4j2.xml).
+
+It will configure the default [log level](https://logging.apache.org/log4j/2.x/manual/customloglevels.html) to be `WARN` for all except for the
+following as `INFO`:
+
+- Camunda 8 (anything under `io.camunda` and `io.atomix`)
+- Spring and Spring Boot (anything under `org.springframework`)
+
+:::note
+[See our documentation on log levels to understand what the various log levels mean.](../../operational-guides/monitoring/log-levels.md)
+:::
+
+You can control the log level for the orchestration cluster via the `CAMUNDA_LOG_LEVEL` environment variable, e.g. `CAMUNDA_LOG_LEVEL=DEBUG`, or
+`CAMUNDA_LOG_LEVEL=ERROR`.
+
+:::note
+You can further use the following environment variables to set the log level for certain components of the application, but we recommend using the general
+`CAMUNDA_LOG_LEVEL` unless you really know what you're doing.
+
+- `ZEEBE_LOG_LEVEL`: will set the level for anything under `io.camunda.zeebe`, i.e. Zeebe related.
+- `ATOMIX_LOG_LEVEL`: will set the level for anything clustering or raft related.
+- `OPTIMIZE_LOG_LEVEL`: will set the level for anything under `io.camunda.optimize`.
```

Comment:
❓ Does it make sense to mention this here? As it is related to zeebe deployment?

Reply:
Good point. I would assume we will remove it? I actually don't know how much of the shared codebase they will reuse, but we could remove it here considering the overall direction :+1: